tests/smoke/helpers.py

```diff
@@ -70,3 +71,66 @@ def _compare_content(
             diff_results.append(line)
 
     return diff_results
+
+
+def _create_everything_uccignore_dir(tmp_dest_dir_name: str) -> None:
+    def __copy_files(
+        source_dir: str, destination_dir: str, ignore_files: List[str] = []
+    ) -> None:
+        # Walk through the source directory
+        for root, dirs, files in os.walk(source_dir):
+            # Create corresponding directories in the destination directory
+            for directory in dirs:
+                source_directory = os.path.join(root, directory)
+                destination_directory = source_directory.replace(
+                    source_dir, destination_dir
+                )
+                if not os.path.exists(destination_directory):
+                    os.makedirs(destination_directory)
+
+            # Copy files to corresponding destination directories
+            for file in files:
+                if file in ignore_files:
+                    continue
+                source_file = os.path.join(root, file)
+                destination_file = source_file.replace(source_dir, destination_dir)
+                copy2(source_file, destination_file)  # copy2 preserves file metadata
+
+    src_path = Path(
```

Comment:
I think it's a step into right direction. We still copy files from `expected_output_global_config_everything` which I would like to avoid.

Would it be possible to create a very minimal version of package for `test_ucc_generate_with_everything_uccignore` test so we do not depend on `expected_output_global_config_everything` at all?

Reply:
Updated the test case to check only the functionality of `.uccignore` file. Rest all functionality are already present in `test_ucc_generate_with_everything`, hence no need to check them again.